bitnami/harbor/templates/core/core-dpl.yaml

```diff
@@ -140,9 +139,6 @@ spec:
           {{- else }}
             secretName: {{ .Values.service.tls.secretName }}
           {{- end }}
-            items:
-              - key: ca.crt
-                path: ca.crt
```

Comment:
Well.. This is removing the `ca.crt` instead of making it optional. Or am I missing anything?

Reply:
I would say that removing `items` we are going to mount just if the ca.crt exist but I will double-check it.

Reply:
**deployment with secret** ca.crt should only appear if the user add it to the secret
```
$ kubectl exec -ti hb-harbor-core-66ccf8c769-97rv4 ls /etc/core/ca
tls.crt  tls.key
```
**deployment without secret** ca.crt should appear always because we are generating it and it is self-signed
```
$k exec -ti hb-auto-harbor-core-7cccbc6c57-wlc6m ls /etc/core/ca
ca.crt  tls.crt  tls.key
```

It seems to be working as expected.